src/planner/mod.rs

```diff
@@ -77,6 +77,7 @@ define_language! {
 
         // plans
         "scan" = Scan([Id; 2]),                 // (scan table [column..])
+        "internal" = Internal([Id; 2]),         // (internal table [column..])
```

Comment:
I prefer not to introduce a new plan node for internal tables, otherwise those optimization rules for scan node can not be applied to internal table. Basically it is the same as `scan`, I think we could reuse the `scan` node until building executors.

Reply:
Got it. I wonder another question: is it acceptable to change the content of `contributors` to author name? I think that if it is acceptable, we can update the `contributors` contents in `build.rs` offline, otherwise `tokio` and `reqwest` may needed to call GitHub RESTful API, and `build-dependencies` may be much larger.

Reply:
I would prefer updating it manually. Otherwise build.rs might be slow and affect the normal build process.

Reply:
@wangrunji0408 but internal table doesn’t support filter push down like the normal storage engine? I guess it’s still a good idea to do it separately… not sure

Reply:
Also internal table doesn’t need to support columns pruning, in my opinion.

Reply:
Sorry I missed this discussion. It seems reasonable to have the internal node. 🆒